v3/docs/01-getting-started/c-dev-env/index.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: Installation
-slug: /v3/getting-started/installation
+title: Development Environment
+slug: /v3/getting-started/dev-env
```

Comment:
@NukeManDan : this type of change needs to be propagated to all other slugs that are intended to point to the same content. 

Additionally I'm unsure about using an "abbrv." in the slug. I would go for something more human readable, like "environment". @Eve-Parity @lsgunnlsgunn thoughts?

Reply:
Fixed, :crossed_fingers: the CI thinks so too. I think `dev` and `env` are effectively industry standard and it helps shorten URLs. Open to find&replace to `developer-environment`